docs/data-cookbooks/cnm-workflow.md

```diff
@@ -0,0 +1,166 @@
+# Basic CNM Workflow
+
+This entry documents setup of a basic workflow that utilizes the built-in CNM/Kinesis functionality in Cumulus.
+
+Prior to using this entry you should be familiar with the [Cloud Notification Mechanism](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cloud+Notification+Mechanism).
+
+## Configuration
+
+### AWS CLI
+
+Use of this entry assumes you have the [AWS CLI](https://www.google.com/search?q=aws+cli) installed and configured.   If you do not, take a moment to review the documentation and install it now.
+
+### Kinesis
+
+Use of this entry assumes you already have a [Kinesis](https://aws.amazon.com/kinesis/) data steam created for use as a CNM notification stream.
```

Comment:
You may want to add something like "In an operational environment, the science data provider would typically be responsible for providing a Kinesis stream with the proper permissions.

Reply:
Great suggestion.     6f3c8e73 .    I'm a bit concerned about the permissions shenanigans in easily setting up a write stream for this example, as it requires at least  Kinesis write policy to be attached to the lambda's role for the write resource (and possibly others in Sandbox, I can't test this scenario against that environment).